src/lib/invention/inventions.ts

```diff
@@ -395,6 +407,51 @@ export const Inventions: readonly Invention[] = [
 		flags: ['equipped'],
 		inventionLevelNeeded: 100,
 		usageCostMultiplier: 3
+	},
+	{
+		id: InventionID.WispBuster,
+		name: 'Wisp-buster',
+		description: `A device increases the speed of memory harvesting, giving ${inventionBoosts.wispBuster.xpIncreasePercent}% more xp.`,
+		item: getOSItem('Wisp-buster'),
+		materialTypeBank: new MaterialBank({
+			pious: 4,
+			powerful: 1,
+			magic: 4,
+			heavy: 1
+		}),
+		itemCost: null,
+		flags: ['equipped'],
+		inventionLevelNeeded: 100,
+		usageCostMultiplier: 1
+	},
+	{
+		id: InventionID.DivineHand,
+		name: 'Divine hand',
+		description: `A device that enhances the harvesting of divination memories, increasing energy yield by ${inventionBoosts.divineHand.memoryHarvestExtraYieldPercent}% and harvesting clue scrolls.`,
+		item: getOSItem('Divine hand'),
+		materialTypeBank: new MaterialBank({
+			pious: 2,
+			magic: 7,
+			strong: 1
+		}),
+		itemCost: null,
+		flags: ['equipped'],
+		inventionLevelNeeded: 100,
+		usageCostMultiplier: 1
+	},
+	{
+		id: InventionID.DrygoreAxe,
+		name: 'Drygore axe',
+		description: `Speeds up woodcutting by ${inventionBoosts.drygoreAxe.woodcuttingSpeedBoostPercent}%.`,
+		item: getOSItem('Drygore axe'),
+		materialTypeBank: new MaterialBank({
+			drygore: 7,
+			sharp: 3
+		}),
+		itemCost: new Bank().add('Dwarven greataxe'),
+		flags: ['equipped'],
+		inventionLevelNeeded: 100,
+		usageCostMultiplier: 0.85
```

Comment:
This should maybe be 0.6 multiplier, like the Drygore saw?

That said, I did some testing, and it's using 18 mats/hr right now, which is 44 hours per drygore ( a bit less with junk chance ).

Side note: drygores/dwarvens probably should have 0% junk chance XD.